Resources/Private/Fusion/Main.scss

```diff
@@ -0,0 +1,18 @@
+// Include libraries
```

Comment:
Why is Main and Variables not in the Styles folder?

Reply:
Because it is the first step for the refactoring. The goal is the get ridden of the Styles folder, that's why it has the same entry point (`Private/Fusion`)

Reply:
But don't we still need a Styles folder for global stuff like variables, typography and such?

Reply:
We never had files directly in Private